DependencyInjection/Compiler/MissingExtensionSuggestorPass.php

```diff
@@ -19,7 +19,7 @@
 class MissingExtensionSuggestorPass implements CompilerPassInterface
 {
     /** @return void */
```

Comment:
perhaps remove this?

Reply:
changed it. Absolutely makes sense, just oversaw it.